external/test-runtime/XUnit.Runtime.depproj

```diff
@@ -37,6 +37,25 @@
 
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
```

Comment:
For some reason, I did not need to add it... Not sure how to tell if I need it or not..